ui/packages/catalog-ui-search/src/main/webapp/react-component/location/base.line.js

```diff
@@ -150,6 +157,23 @@ class BaseLine extends React.Component {
       return false
     }
   }
+  convertWkt(value, numCoords) {
+    const coordinates = value.match(coordinatePairRegex)
+    if(!coordinates || coordinates.length < numCoords) {
+      return value
+    }
+    let newValue = "[["
+    for(let i = 0; i < coordinates.length; i++) {
+      const coordinatePair = coordinates[i].split(" ")
+      newValue = newValue + coordinatePair[0] + "," + coordinatePair[1]
+      if(i === coordinates.length - 1) {
+        newValue = newValue + "]]"
+      } else {
+        newValue = newValue + "],["
+      }
+    }
+    return newValue
```

Comment:
:pencil2: Could do this in a map and join instead

```
const newCoords = coordinates.map(coord=> coordinates[i].split(" ").join(','))
return `[[${newCoords.join('],['}]]`
```

Reply:
Oh sweet, thanks!